java/code/src/com/redhat/rhn/manager/audit/test/ScapManagerTest.java

```diff
@@ -326,9 +317,6 @@ public void testXccdfEvalTransformXccdf12() throws Exception {
                 .openStream();
         XccdfTestResult result = ScapManager.xccdfEval(minion, action, 2, "", resultsIn, resumeXsl);
 
-        HibernateFactory.getSession().flush();
-        HibernateFactory.getSession().clear();
```

Comment:
not sure if we get new data from DB when we remove flush and clear.
Is it not working if we keep it?

Reply:
My impression was that with the refactoring I introduced some `CascadeType.ALL` that prevents some `flush` ...but also in this case it has sense to verify if it's really required or not.

Reply:
re-adding causes
 ```
java.lang.UnsupportedOperationException: Can't update readonly object
```
in 
```
result = HibernateFactory.getSession().get(XccdfTestResult.class, result.getId());
```
I think it doesn't require further analysis and better to remove them...what do you think?

Reply:
I think you should write down that Xccdf  should be tested in "real life".
I this this is somehow caused by using `org.hibernate.annotations.CacheConcurrencyStrategy.READ_ONLY` .
So better you install an instance after it is merged and test OpenSCAP scanning and viewing the results.
If that is ok we are fine.
You can remove this for now.